css/style.css

```diff
@@ -762,18 +774,20 @@ li.consistency-header-index {
     box-sizing: border-box;
 }
 
-.gd-on-translations .gp-content .bulk-actions input.button,
+#bulk-actions-toolbar-top{
+	float:left !important;
+}
+
 .gd-on-translations .gp-content input.button.gd-review {
-    vertical-align: top;
-    height: auto;
     margin: 0 5px;
-    padding: 10px;
-    font-size: 14px;
+}
 
+#gd-sticky-header .paging{
+	width: 100%;
```

Comment:
Can we replace `width: 100%` with `float: left` ? to perserve the original layout?

Reply:
If it keeps the same design of course, I thought that I tried but that it was breaking something. But if you tell that it's good, go on :)